processes.py

```diff
@@ -203,10 +204,10 @@ def process_to_dict(process):
       'Prepare to ship',
       'Lock in shipping milestone. Finalize docs and announcements. '
       'Further standardization.',
-      ['Intent to Ship email',
-       'Three LGTMs',
-       'Documentation signoff',
+      ['Documentation signoff',
        'Updated target milestone',
+       'Intent to Ship email',
+       'Three i2s LGTMs',
```

Comment:
```suggestion
       'Three LGTMs for the intent to ship',
```

Reply:
Do we have room to spell this out? Abbreviations of any kind are barriers to new participants, even if they seem obvious to us.

Reply:
If we have the room, I can change the others, but won't object if you get there first.

Reply:
Done.